docs/view_components/energy_system_network.rst

```diff
@@ -33,84 +35,88 @@ Attributes
     Linked to the action of removing a bus from the `es_bus_list`
 
     Properties:
-        - id `remove_es_bus`
+        * id `remove_es_bus`
 
     Actions index: 2, 3
 
     Requirements index:
 
 **list of energy system component**
-    properties:
+    A list of *energy system components*
+    
+    Properties:
         - id: `es_component_list`
-        - each energy system component has a unique id
-        - each energy system component has a list of associated :ref:`energy-type-def`
-        - each energy system component can have in and/or out connection to one of the energy busses or one of the other energy system components
-        - each energy system component has a type (sink, source, transformer, storage)
+        - each *energy system component* has a unique id
+        - each *energy system component* has a list of associated :ref:`energy-type-def`
+        - each *energy system component* can have in and/or out connection to one of the energy busses or one of the other energy system components
+        - each *energy system component* has a type (sink, source, transformer, storage)
+        .. these properties are all described in energy_system_component.rst already, isn't that redundant? Should we delete them here?
```

Comment:
Indeed, then we could put this category in *link with other view components*

Reply:
Sorry I see now I had a review started and I forgot to press "submit review" ...

Reply:
okay, the link with view-component *energy system component* does already exist, so i close both comments.